src/zcbenchmarks.cpp

```diff
@@ -465,3 +465,12 @@ double benchmark_loadwallet()
     post_wallet_load();
     return res;
 }
+
+double benchmark_listunspent()
+{
+    UniValue params(UniValue::VARR);
+    struct timeval tv_start;
+    timer_start(tv_start);
+    auto unspent = listunspent(params, false);
```

Comment:
Local variable unspent is not used... do we need to store the result of calling listunspent?

Reply:
Ah, no we don't need to, but unfortunately GitHub didn't show me this feedback until *after* I r+ed the PR. Looks like there's at least one other instance of this however, so I'll make a follow-up cleanup PR.